docs/guide/deployer.md

```diff
@@ -63,6 +89,37 @@ The main difference between deploying an ASA and SSC is that ASA takes `asset-na
 You can learn more about the flags from [Deployer API](https://scale-it.github.io/algo-builder/);
 You can learn more about Stateful Smart Contracts [here](https://developer.algorand.org/docs/features/asc1/stateful/).
 
+#### OptIn to SSC
+
+For opting in to SSC, `deployer` supports two methods :-
+- `optInAcountToSSC` to opt-in to a single account signed by secret key of sender.
+- `optInLsigToSSC` to opt-in to a contract account (say escrow) where the account is represented by the logic signature address (`lsig.address()`).
+  - To opt in to SSC you can use `Application Index`.
+
+- There is one more method which you can use to opt-in, It can be used with group transactions also
+- `executeTransaction` to opt-in single account or contract account to SSC.
+- Ex: To opt-in a single account, Params will look like this:
+```js
+   const execParam: ExecParams = {
+    type: TransactionType.OptInSSC,
+    sign: SignType.SecretKey,
+    fromAccount: user.account,
+    appID: appID,
+    payFlags: payFlags
+  };
+```
+- Ex: To opt-in to a contract account
+```js
+   const execParam: ExecParams = {
+    type: TransactionType.OptInSSC,
+    sign: SignType.LogicSignature,
+    fromAccountAddr: lsig.address(),
+    appID: appID,
+    payFlags: payFlags
+    lsig: lsig
+  };
+```
```

Comment:
same as above. Maybe we can reword it, a bit? 

"Like with ASA, we can also use `executeTransaction` to opt-in a single account or contract account to SSC. Example: ..."

Reply:
ok